tools/db_bench_tool.cc

```diff
@@ -8185,6 +8213,49 @@ class Benchmark {
     }
   }
 
+  void Backup(ThreadState* thread) {
+    std::cout << "Backup path: [" << FLAGS_backup_dir << "]" << std::endl;
```

Comment:
Is there a way to move the printing outside of the benchmarked function - detto for Restore() below? 

I am thinking of maybe something before https://github.com/facebook/rocksdb/blob/7.2.fb/tools/db_bench_tool.cc#L3516-L3518

Reply:
Do we want to print them even when backup/restore benchmarks are not ran? I currently put them inside Backup()/Restore() so that they don't get print out in other benchmarks. 

I do realize that I should at least print Backup Path **and** Restore Path for the restore benchmark tho.

Reply:
Can we conditionally print the path based on the `name` of the benchmark being run? I see an example like this https://github.com/facebook/rocksdb/blob/7.2.fb/tools/db_bench_tool.cc#L3451

Reply:
Sg, updated.